TFT/src/User/Menu/Babystep.c

```diff
@@ -25,14 +28,17 @@ void babyReDraw(float babystep, float z_offset, bool force_z_offset, bool skip_h
   setFontSize(FONT_SIZE_LARGE);
   sprintf(tempstr, "% 6.2f", babystep);
   GUI_DispStringRight(point_bs.x, point_bs.y, (uint8_t*) tempstr);
-  sprintf(tempstr, "% 6.2f", z_offset);
+  if (infoMachineSettings.firmwareType != FW_REPRAPFW)
+  {
+    sprintf(tempstr, "% 6.2f", z_offset);
 
-  if (force_z_offset)
-    GUI_SetColor(infoSettings.reminder_color);
-  else
-    GUI_SetColor(infoSettings.font_color);
+    if (force_z_offset)
+      GUI_SetColor(infoSettings.reminder_color);
+    else
+      GUI_SetColor(infoSettings.font_color);
 
-  GUI_DispStringRight(point_of.x, point_of.y, (uint8_t*) tempstr);
+    GUI_DispStringRight(point_of.x, point_of.y, (uint8_t*) tempstr);
+  }
```

Comment:
please add an empty line after the `if` block.

Reply:
fixed